pkgs/applications/networking/instant-messengers/gajim/default.nix

```diff
@@ -1,7 +1,16 @@
 { buildPythonApplication, lib, fetchurl, gettext, wrapGAppsHook
+
+# Native dependencies
 , python, gtk3, gobject-introspection
-, nbxmpp, pyasn1, pygobject3, gnome3, dbus-python, pillow
+
+# Python dependencies
+, nbxmpp, pyasn1, pygobject3, gnome3, dbus-python, pillow, cssutils
+, precis-i18n, keyring
```

Comment:
Would you mind getting rid of `python3Packages.callPackage` in `all-packages.nix` and specifying `python3` as argument instead?